cmd/schema-gen/model/model.go.txt

```diff
@@ -91,6 +91,42 @@ type (
 		Source string `json:"source"`
 	}
 
+	JSONSchema struct {
+		Schema string `json:"$schema,omitempty"`
+
+		Ref string `json:"$ref,omitempty"`
+
+		Definitions map[string]JSONSchema `json:"definitions,omitempty"`
+
+		Type string `json:"type,omitempty"`
+
+		Format string `json:"format,omitempty"`
+
+		Pattern string `json:"pattern,omitempty"`
+
+		Properties map[string]JSONSchema `json:"properties,omitempty"`
+
+		Items JSONSchema `json:"items,omitempty"`
+
+		Enum []any `json:"enum,omitempty"`
+
+		Required []string `json:"required,omitempty"`
+
+		Description string `json:"description,omitempty"`
+
+		Minimum float64 `json:"minimum,omitempty"`
+
+		Maximum float64 `json:"maximum,omitempty"`
+
+		ExclusiveMinimum bool `json:"exclusiveMinimum,omitempty"`
+
+		ExclusiveMaximum bool `json:"exclusiveMaximum,omitempty"`
+
+		MultipleOf float64 `json:"multipleOf,omitempty"`
+
+		AdditionalProperties any `json:"additionalProperties,omitempty"`
+	}
```

Comment:
Didn't cover with godoc. Add some comments or reduce spacing if no comments on field. Give the type itself a comment (must).